x/evm/types/config.go

```diff
@@ -44,7 +44,7 @@ func (cc ChainConfig) EthereumConfig(chainID *big.Int) *params.ChainConfig {
 
 func DefaultChainConfig() ChainConfig {
 	return ChainConfig{
-		CancunTime: -1,
+		CancunTime: 0,
```

Comment:
why is this changed?

Reply:
cancun will be launched on eth mainnet  on 03/13, so we've decided to also support cancun version (given our mainnet launch would be after that). See https://sei-network-io.slack.com/archives/C069MR5S82Z/p1707752152756309?thread_ts=1707751607.560409&cid=C069MR5S82Z for context